### PR TITLE
[DR-1421] Prevent multiple clicks on upgrade button

### PR DESCRIPTION
**Pull request by alegarciamina**
Disable submit button while processing the upgrade request.

### CODE REVIEW

File: src/wwwroot/controllers/BillingCtrl.js [hunk 1]
```diff
@@ -176,7 +178,7 @@
         downgrade();
         return;
       }
-
+
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: Very minor.  Remove this change.

File: src/wwwroot/controllers/BillingCtrl.js [hunk 2]
```diff
@@ -153,6 +154,7 @@
     }
 
     var onExpectedError = function (rejectionData) {
+      vm.processingPayment = false;
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: What do you think about using _finally_ (or something like that)

**Comment by andresmoschini** [comment]
:+1:

File: src/wwwroot/controllers/BillingCtrl.js (hunk 2 above)
Comment (alegarciamina) [reply src/wwwroot/controllers/BillingCtrl.js]: User is redirected to My Plan on success, so it's not strictly necessary but finally looks cleaner and self-descriptive. I'll change it.

**Pull request merged**